### ISSUE TITLE
each binder using incorrect model reference after upgrade from 0.6 to 0.8

### ISSUE THREAD

**Issue by tanshul22** [mikeric/rivets#616]
I'm building an enterprise application which deals with large Collections. These Collections at times go through a major revision ie. their contents change dramatically. In such cases, I try to prevent multiple add/remove/sort events from triggering rivets updates by using a Collection reset instead. I've demonstrated this strategy in the JSFiddle below. This strategy has worked great up until recently when we upgraded from rivets 0.6.10 to 0.8.1. I've read the change-log thoroughly and can't find any changes in this regard.

A lot of time has been put towards upgrading our version of rivets but unfortunately we won't be able to go through with the new changes until I can address this bug. Any help is much appreciated.

[https://jsfiddle.net/wvc3wp0k/2/](https://jsfiddle.net/wvc3wp0k/2/)

Thank you!

**Comment by tanshul22** [comment]
I've created a non-Backbone JSFiddle if it helps.

[https://jsfiddle.net/s2qags6h/](https://jsfiddle.net/s2qags6h/)

**Comment by jccazeaux** [comment]
Duplicate with #572

**Comment by tanshul22** [comment]
Thanks for replying @jccazeaux. You're right, it does seem like #572. Is my proposed fix the actual fix or is it just a workaround?

```
    // ak: Fix for issue #616 - sync after updating models
    TextBinding.prototype.update = function() {
      var _results = [];
      _results.push(TextBinding.__super__.update.apply(this, arguments));
      _results.push(TextBinding.__super__.sync.apply(this, arguments));
      return _results;
    };
```
